scripts/README.md

```diff
@@ -2,20 +2,22 @@
 
 This repository contains scripts that are used by multiple release steps.
 
+<b>Note:</b> Make sure you have a `~/.my.cnf` file populated with `mysqldump` and `mysql` parameters to use this script. See https://stackoverflow.com/a/24369804
 
-<h3>updateDatabase.sh</h3>
 
-<b>Purpose</b>: Create or replace a mySQL database with a mysql dump file.
+<h3>createOrReplaceDatabase.sh</h3>
+
+<b>Purpose</b>: Create or replace a MySQL database with a mysql dump file.
 <br>
 <b>Perl version</b>: <a href="https://github.com/reactome/Release/blob/master/scripts/restore_database.pl">restore_database.pl</a>
 <br>
 <b>Arguments</b>:
-- `-f|--file`  Filepath of mySQL dump file that will be restored
-- `-d|--database`  Name of mySQL database that will be created/updated
-- `-c|--config`  (Optional) Filepath of config file that contains mySQL parameters (username, password, database, port)
+- `-f|--file`  Filepath of MySQL dump file that will be restored
+- `-d|--database`  Name of MySQL database that will be created/updated
+- `-c|--config`  (Optional) Filepath of config file that contains mySQL parameters (username, database, port)
```

Comment:
If password isn't taken from the config file, it may be better not to allow username to be changed either?  Both should probably be configured in the ".my.cnf" file.

Reply:
Yea I thought to do that but wasn't sure if there was some situation where you might want to be a different user... but that doesn't make sense if your using a specific user's password, so I'll make that change

Reply:
Done